phpcs.xml

```diff
@@ -81,6 +81,12 @@
 	<arg name="extensions" value="php"/>
 	<file>.</file>
 
+	<!-- Strip the filepaths down to the relevant bit. -->
+	<arg name="basepath" value="./"/>
```

Comment:
I wonder if this is causing the Travis build failure.

Reply:
The error comes from Git, even before any PHPUnit or PHPCS is run.

It usually happens when a branch gets deleted before the tests are run. Not sure why it happens in this case.

Given that the issue only happens for the merge commit / PR build (c51fa7940e7021fdad9808c502497e31bf30f782) and the tests run fine for the push build, it's more of a glitch on Travis side than something in this PR.